constellations/gemini-admin/src/classes/classEdit.js

```diff
@@ -294,6 +308,72 @@ export class ClassEditPage extends React.Component {
                         <option value="1">Almost Full</option>
                         <option value="2">Full</option>
                     </select>
+
+                    <InputText
+                        label="Google Classroom Code"
+                        isTextBox={true}
```

Comment:
should not be a textBox (just a single number for input). Textboxes are for large texts like paymentNotes.